examples/src/home.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export function Home() {
+    return (
+        <>
+            EXAMPLES
+            <br />
+            <ul>
+                <li>
+                    <a href="/dzi">Deep Zoom Image</a>
+                    <br />
+                </li>
+                <li>
+                    <a href="/omezarr">OMEZARR</a>
+                    <br />
+                </li>
+                {/* layers is not in the AC to be converted to a react component
```

Comment:
Let's at least let it render like it was, since there's a good Markdown page explaining how to use it in the docs still so we want it working.